packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -355,12 +358,12 @@ class PopupMenuItemState<T, W extends PopupMenuItem<T>> extends State<W> {
         child: item,
       );
     }
-    final MouseCursor effectiveMouseCursor = MaterialStateProperty.resolveAs<MouseCursor>(
-      widget.mouseCursor ?? MaterialStateMouseCursor.clickable,
-      <MaterialState>{
-        if (!widget.enabled) MaterialState.disabled,
-      },
-    );
+    final Set<MaterialState> states = <MaterialState>{
```

Comment:
InkWell already resolves the mouseCursor against focused/hovered/pressed/disabled and uses MaterialStateMouseCursor.clickable when mouseCursor is null. It looks like the right thing to do here is to just pass the mouse cursor along.